ape_ens/converter.py

```diff
@@ -20,34 +39,25 @@ def is_convertible(self, value: Any) -> bool:
         elif not ENS.is_valid_name(value):
             return False
 
+        elif value in self.address_cache:
+            return True
+
         else:
             try:
-                with self._connect_to_ens() as ens:
-                    return ens.address(value) is not None
+                address = self.mainnet_provider.web3.ens.address(value)
```

Comment:
Put a little TODO that eventually we want the ENS SDK available to perform lookups with